### PR TITLE
Pr to merge feature branch into development

### PR DESCRIPTION
**Pull request by talhawaqar**
This pr include the complete weather app. This weather app uses OpenWeatherMap API to display weather data to user.
There are the following aims of building this project.

-  How to use API using simple plain js
- Learn about async and await functions

### CODE REVIEW

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi @talhawaqar :hand: 
Great job on this app so far :+1: 
But we need to make some changes before moving on.

- [x] Your .eslintrc file is different from the one provided by Microverse. We need to use this configuration [instead](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json).

- [x] I liked your idea to show the flag's country from the inputted location. However, since this is a weather app, we still need to change something in the UI related to the current weather. As the requirement said, you can change the color of the background, to a color that reflects the current weather, or you can add a corresponding icon.
Screenshot from 2021-01-15 13-09-23

Feel free to message me on Slack if you have questions.
Happy coding :computer: 
**Sergio Torres**

File: src/dom.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { weatherInfo, getImage } from './api-requests';
+
+const widgetDiv = document.getElementById('weather-info');
+
+const displayWeather = async (city, centigradeUnit) => {
+  const weather = await weatherInfo(city);
+  let unit = '';
+  let temperature = 0;
+  let feelLike = 0;
+  // console.log(weather.weather[0]);
```
Comment (whiz25) [inline src/dom.js]: - remove this comment and any other comments in the codebase.

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required :pray:
Nice work on the requested changes. The background image is fixed and `.eslintrc.json` is updated as requested, however, you have a comment in `dom.js` file. Please remove it as it makes your code ugly.

### Martin Afani

**Review by elmejdki — APPROVED** [review]
# status: approved :heavy_check_mark: 

Great job @talhawaqar :clap: :clap: :clap:, and thanks for making the required changes :hammer: by the previous TSE's

Now you can merge :green_circle: your project is approved :checkered_flag: 

## OPTIONAL:

- [ ] Now your project is approved, but please I have one request, add a live demo link in the readme.

## Happy Coding :tophat:

**Pull request merged**